pep-9999.rst

```diff
@@ -13,9 +13,47 @@ Post-History:
 Abstract
 ========
 
+Optional type hints were introduced to the Python language in PEP 484
+[#pep484]_, based on the function annotation syntax from PEP 3107
+[#pep3107]_. PEP 484 also introduced *type stubs*, files that provide type
+information for untyped Python packages and modules. Type stubs serve
+multiple purposes:
+
+* They are the only way to add type information to extension modules.
+* They can provide type information for packages that do not wish to
+  add them inline.
+* They can be distributed separately from the implementation.
+  This allows stubs to be develped at a different pace or by different
+  authors, which is especially useful when adding type annotations to
+  existing packages.
+* They act as documentation, succinctly explaining the external
+  API of a package, without including the implementation or private
+  members.
+
+This PEP aims to give guidance to both authors of stub files and developers
+of type checkers. It lists constructs that compliant type checkers are
```

Comment:
Based on this sentence, on my first read-through, I (1) expected the next paragraph to be about compliant type checkers, and (2) hadn't yet been introduced to the concept that type stubs can also be compliant, so the transition to the next paragraph was a little jarring. I'd flip the two halves of this sentence and add an extra "compliant" in there: "It gives guidance on the contents and style of *compliant* stub files and lists constructs that compliant type checkers are expected to support."

Reply:
I needed to reshuffle this whole paragraph a bit to avoid "giving guidance" twice. Maybe we could also drop the word "compliant" from this paragraph completely?

Reply:
I think keeping "compliant" makes the transition to the next paragraph a little smoother, but I don't feel strongly about this.